src/main.C

```diff
@@ -33,7 +33,7 @@ main(int argc, char * argv[])
   BlackBearTestApp::registerApps();
 
   // register inelastic strain
-  TensorMechanicsActionBase::addCartesianComponentOutput("inelastic_strain");
```

Comment:
I think you don't want this change, right?

Reply:
No, I don't want these changes.  I will commit my changes again to include this one and one for the input file for the "sulfide_attack.i"

Reply:
Ben, It seems like I still need to change several other files in order to build Blackbear in this branch. I am not sure why but please do not merge it.